sympy/parsing/autolev/Autolev.g4

```diff
@@ -0,0 +1,111 @@
+grammar Autolev;
+
+prog:   stat+;
+
+stat:   varDecl
+|       functionCall
+|       codeCommands
+|       massDecl
+|       inertiaDecl
+|       assignment
+|       settings
+;
+
+assignment:     vec ('='|'+='|'-='|':='|'*=') expr #vecAssign
+|               ID '[' index ']' ('='|'+='|'-='|':='|'*='|'/='|'^=') expr #indexAssign
+|               ID diff? ('='|'+='|'-='|':='|'*='|'/='|'^=') expr #regularAssign; 
+
+index: expr (',' expr)* ;
+
+diff: ('\'')+;
+functionCall:   ID '(' (expr (',' expr)*)? ')'
+|               (Mass|Inertia) '(' ID (',' ID)* ')'; 
+
+varDecl:   varType varDecl2
+(',' varDecl2)*;
+
+varType: Newtonian|Frames|Bodies|Particles|Points|Constants|Specified
+|Specifieds|Imaginary|Variables ('\'')*|MotionVariables ('\'')*;
+
+varDecl2: ID (('{' INT ':' INT (',' INT ':' INT)* '}'))? 
+('{' INT '}')? ('+'|'-')? ('\'')* ('=' expr)?;
+
+ranges: ('{' INT ':' INT (',' INT ':' INT)* '}');
+
+massDecl: Mass massDecl2 (',' massDecl2)*;
+
+massDecl2:  (ID '=' (INT|FLOAT)|ID '=' ID|ID '=' ID '=' (INT|FLOAT));
+
+inertiaDecl: Inertia ID (',' expr)+;
+
+matrix: '[' expr ((','|';') expr)* ']';
+matrixInOutput: (ID (ID '=' (FLOAT|INT)?))|FLOAT|INT;
+
+codeCommands:  units
+|              inputs
+|              outputs
+|              codegen
+|              commands;
+
+settings:   ID (EXP|ID|FLOAT|INT)?;
+
+units: UnitSystem ID (',' ID)*;
+inputs: Input inputs2 (',' inputs2)*;
+inputs2: ID('\'')* '=' expr expr?;
+outputs: Output outputs2 (',' outputs2)*;
+outputs2: ID('\'')* expr?;
+codegen: ID functionCall ('['matrixInOutput (',' matrixInOutput)*']')? ID'.'ID;
+commands: Save ID'.'ID
+|         Encode ID (',' ID)*;
+
+vec:  ID ('>')+
+|      '0>'
+|      '1>>';
```

Comment:
In here, please add spaces before "|", so that it is aligned with the ":", just like you have done for the `expr:` below. I think that seems to be the accepted formatting for the g4 files.

Reply:
And the same for things above.